app/models/daily_usage_creation/steps/appliance.rb

```diff
@@ -3,8 +3,54 @@
 module DailyUsageCreation
   module Steps
     class Appliance < WizardSteps::Step
-      attribute :id, :string
-      attribute :quantity, :integer
+      attribute :appliance_id, :string
+      attribute :quantity, :integer, default: 1
+
+      validates :appliance_id, presence: { message: "Select an appliance from the list" }
```

Comment:
Is it worth adding a couple of specs to cover the error messaging?

Reply:
I was going to do this as part of the feature tests, I feel like the existing `#valid?` tests are probably enough here (especially since it's not a custom validator and it's OOTB rails.  But happy to add them if you think I should?

Reply:
Nah, that sounds sensible 🙂